ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -267,10 +267,29 @@ public struct Avatar: View, ConfigurableTokenizedControl {
             }
         }
 
+        let standardAnimation = Animation.linear(duration: animationDuration)
+
         return avatarBody
             .pointerInteraction(state.hasPointerInteraction)
             .modifyIf(state.isAnimated, { thisView in
-                thisView.animation(.linear(duration: animationDuration))
+                thisView
+                    .animation(standardAnimation,
```

Comment:
Is there a cost impact to having six calls to animation rather than just one? Is there any way to condense this?

Reply:
This call is not the one responsible for performing the animation itself. It's just setting it up based on the properties passed.

Previously the way to condense all of them was to just have a call into the animation modifier with no value specified. It'd apply to any changes in the SwiftUI view. Apple just deprecated this call and now we need to have explicit calls to all properties that will cause the view to animate.